src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -395,13 +401,30 @@ private Slave prepareDockerPlatforms() throws Exception {
         c.cleanup(ws);
     }
 
+    /**
+     * Checks if the golang binary outputs to stdout under normal shell execution.
+     * The binary must NOT output to stdout or else it will crash when Jenkins is terminated
```

Comment:
Is this still relevant now that the binary runs in the background?

Reply:
You are probably right, however if you choose not to run it in `daemon` mode (ie. `LAUNCH_DIAGNOSTIC = true`, then I guess it would still be a problem? But I'm not certain what the normal use cases of `LAUNCH_DIAGNOSTIC` is.

Reply:
That's fair. I think `LAUNCH_DIAGNOSTICS` is essentially unused outside of troubleshooting, because it has to be set manually by an admin.